packages/@react-types/shared/src/collections.d.ts

```diff
@@ -130,6 +130,9 @@ export interface Collection<T> extends Iterable<T> {
   /** Get an item by its key. */
   getItem(key: Key): T,
 
+  /** Get an item by the index of its key. */
+  getAt(idx: number): T,
```

Comment:
Missed the rename of this one?

Reply:
My bad. Just fixed. Next time if this happens you guys can open a PR. This is prob why open sourcing is annoying to the more experienced ppl 😂

Reply:
Not a problem, we are here to help! Thanks for the PR